app/models/hike.rb

```diff
@@ -60,7 +60,7 @@ def simplified_waypoints(array)
   end
 
   def average_rating
-    reviews.count > 0 ? (reviews.sum(:rating).to_f/reviews.count).round(0) : 0
+    reviews.count > 0 ? reviews.average(:rating) : 0
```

Comment:
This could be further simplified to:

```ruby
reviews.average(:rating) || 0
```

Because when there are no records, `average` will return `nil`.